src/etc/inc/plugins.inc.d/vxlan.inc

```diff
@@ -109,15 +111,17 @@ function vxlan_configure_do($verbose = false, $device = null)
             $isChanged = false;
             $current_settings['vxlanid'] = $interfaces_details[$device_name]['vxlan']['vni'];
             $current_settings['vxlanlocal'] = explode(":", $interfaces_details[$device_name]['vxlan']['local'])[0];
+            $current_settings['vxlanlocalport'] = explode(":", $interfaces_details[$device_name]['vxlan']['localport'])[0];
```

Comment:
This looks suspicious. The port should be at [1] of the previous “:” explode() line.